### PR TITLE
build: fix bundlewatch diff report

### PR DESCRIPTION
**Pull request by josokinas**
## Purpose

Following https://github.com/onfido/castor/pull/699 fix bundle size diff.

## Approach

Tracking all `dist` JS and CSS files (of all packages) and prototypes, also setting `main` branch as base and the one to track.

In addition, setting max file sizes (subject to be reviewed), renaming script as "inspect".

## Testing

Report on PR.

## Risks

Not 100% sure diff will work, will need to be merged first in order to take effect.

### CODE REVIEW

**Comment by josokinas** [comment]
Due to an issue with Bundlewatch [erroring on large file number watch when generating URL](https://github.com/bundlewatch/bundlewatch/issues/34) I'm shortening list of files to only base CSS and prototypes.

File: .bundlewatchrc.json [hunk 1]
```diff
@@ -1,17 +1,16 @@
...
+  },
   "files": [
-    {
-      "path": "./packages/core/dist/index.js",
-      "maxSize": "200B"
-    },
     {
       "path": "./packages/core/dist/castor.css",
-      "maxSize": "3kB"
+      "maxSize": "5KB"
```
Comment (rabelloo) [inline .bundlewatchrc.json:9]: Should we watch for a theme as well?

**Review by rabelloo — APPROVED** [review]

File: .bundlewatchrc.json (hunk 1 above)
Comment (josokinas) [reply .bundlewatchrc.json:9]: Theme consists of theme tokens, same as for Castor Tokens package I don't think we have much control over it.

Comment (rabelloo) [reply .bundlewatchrc.json:9]: Fair enough.

**Pull request merged**